backend/dataall/core/permissions/services/organization_permissions.py

```diff
@@ -18,4 +18,4 @@
     INVITE_ORGANIZATION_GROUP,
     REMOVE_ORGANIZATION_GROUP,
 ]
```

Comment:
`REMOVE_ORGANIZATION_GROUP,` is not used, I would avoid dead code

Reply:
it's used in `organization_service.py` , line 210
`    @staticmethod
    @TenantPolicyService.has_tenant_permission(MANAGE_ORGANIZATIONS)
    @ResourcePolicyService.has_resource_permission(REMOVE_ORGANIZATION_GROUP)
    def remove_group(uri, group):`

If I accidentally added the wrong group I would want to delete it as well.

Reply:
I meant, we can use INVITE_GROUP, but it is ok I guess to have 2 permissions. I did not want to have this permission added here because we might want to introduce `REMOVE_ORGANIZATION_GROUP` relative to groupUri not organizationUri on the future

Reply:
I can do the following
1. Rename INVITE_ORGANIZATION_GROUP -> MANAGE_ORGANIZATION_GROUP
2. Make database migration for it
3. Use it for both add and remove groups